tap_sftp/client.py

```diff
@@ -108,6 +109,19 @@ def get_files_by_prefix(self, prefix):
                 if is_empty(file_attr):
                     continue
 
+                # skip gzip file if it is empty
```

Comment:
@harshpatel4crest Please write unit test cases for the same

Reply:
Done